api/turing/service/ensembling_job_service_test.go

```diff
@@ -16,13 +16,18 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	artifactFolder string = "artifact"
```

Comment:
It's there because in the unit test i defined the bucket to be `gs://bucket/artifact`. It's not necessarily `artifact` https://github.com/ashwinath/turing/blob/fix-ensembler-orchestration/api/turing/service/ensembling_job_service_test.go#L343, it's defined in the mock service.

Reply:
This value is populated when you create an ensembler, and it plucks the value from the GS bucket URI.